src/main/java/club/sk1er/patcher/mixins/bugfixes/VertexBuffer_ResolveLockups.java

```diff
@@ -0,0 +1,22 @@
+package club.sk1er.patcher.mixins.bugfixes;
+
+import net.minecraft.client.renderer.vertex.VertexBuffer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.nio.ByteBuffer;
+
+@Mixin(VertexBuffer.class)
+public abstract class VertexBuffer_ResolveLockups {
+    @Shadow private int glBufferId;
+
+    @Inject(method = "bufferData", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/renderer/vertex/VertexBuffer;bindBuffer()V"), cancellable = true)
```

Comment:
This can be `@At("HEAD")`

Reply:
This was original `@At("HEAD")` but if it's there it will break OptiFine's Render Regions; OptiFine makes modifications here and does something else if Render Regions are enabled.

Reply:
Please add a comment indicating that is the case.